src/main.py

```diff
@@ -250,13 +250,16 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
                 video = utils.create_video_from_partial_refinements(
                     e.info["partial_refinements"], env, "test", test_task_idx,
                     CFG.horizon)
-                outfile = f"{video_prefix}__task{test_task_idx+1}_failure.mp4"
+                outfile = f"{video_prefix}__task{test_task_idx+1}.mp4"
                 utils.save_video(outfile, video)
             continue
         solve_time = time.time() - solve_start
         num_found_policy += 1
+        log_message = None
```

Comment:
do you really have to initialize this out here? all code paths should lead to it being a string right? and we want to crash later if that's not the case? we wouldn't ever want "Task 4 / 50: None"